webapi/src/main/scala/org/knora/webapi/app/ApplicationActor.scala

```diff
@@ -521,15 +516,12 @@ class ApplicationActor extends Actor with Stash with LazyLogging with AroundDire
 
   override def postStop(): Unit = {
     super.postStop()
-    logger.info("ApplicationActor - shutdown in progress, initiating post stop cleanup. Bye!")
-
-    if (knoraSettings.prometheusEndpoint) {
-      // Stop Kamon monitoring
-      Kamon.stopModules()
-    }
+    logger.info("ApplicationActor - ... shutdown in progress, initiating post stop cleanup...")
```

Comment:
I have no experience with logging in Scala, but from the Python world, it seems to me, that the logger should know where it is, so it should imo. not be necessary to tell it explicitly that we are in `ApplicationActor` here. Maybe we can discuss this some time - I'm generally a bit unclear about our logging conventions, and what may need change there as well.